core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -125,6 +125,7 @@
 import java.util.stream.Stream;
 
 import static org.apache.kafka.server.log.remote.metadata.storage.TopicBasedRemoteLogMetadataManagerConfig.REMOTE_LOG_METADATA_COMMON_CLIENT_PREFIX;
+import static org.apache.kafka.server.log.remote.storage.RemoteLogManagerConfig.*;
```

Comment:
I think checkstyle hates star imports :)

Reply:
Ah, initially there were individual imports, but I manually changed to wildcard. Fixed now.